### DISCUSSION TITLE
Phone Heat

### DISCUSSION THREAD

**Discussion by jeffcdo** [BLeeEZ/amperfy#263] [Q&A] [answered]
I'm concerned my phone is running so warm with Amperfy. Is there anything I can do to mitigate this? Since the album and song counts are still (very slowly) incrementing, will completing it's initial scan help this?

**Comment by BLeeEZ** [comment, marked as answer]
When the song background sync is finished, the heavy database update load is done and the heat issue should be resolved.

**Comment by BLeeEZ** [comment]
I heavily optimised the performance of all backend APIs. With v1.2.3. the heat should be gone or only after the initial server login/resync.
